source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,534 @@ Filters Builders
    :class: singlecol
 
 .. _filters-builders:
+
+Overview 
+--------
+In this guide, we show you how to use **builders** to specify
+**filters** for your queries.
```

Comment:
I think we should point out some of the places where filters are actually used, e.g.

* find
* match stage of aggregate
* findAndDelete/findAndUpdate/findAndReplace
* countDocuments
* distinct
* deleteOne/deleteMany
* updateOne/updateMany
* replaceOne